model/src/form/form-submission/enums.ts

```diff
@@ -0,0 +1,30 @@
+export const securityQuestions = [
```

Comment:
```
export enum SecurityQuestionEnum {
  MemorablePlace = 'memorable-place',...
}
```
?

Lose the presentation strings ('What is a memorable place you have visited?') as these should probably be in the runner.